golemhost/retrypolicy.go

```diff
@@ -0,0 +1,47 @@
+package golemhost
+
+import (
+	"time"
+
+	binding "github.com/golemcloud/golem-go/golem_go_bindings"
+)
+
+type RetryPolicy struct {
+	MaxAttempts uint32
+	MinDelay    time.Duration
+	MaxDelay    time.Duration
+	Multiplier  uint32
```

Comment:
This should have jitter too now

Reply:
But it's not in the WIT definition yet. However multiplier is now `f32`:
https://github.com/golemcloud/golem-wit/blob/main/wit/deps/golem/golem-host.wit#L49